pkg/virtctl/guestfs/guestfs.go

```diff
@@ -162,17 +161,17 @@ func init() {
 	SetDefaultImageInfoGetFunc()
 }
 
-func (c *guestfsCommand) run(cmd *cobra.Command, args []string) error {
-	pvc = args[0]
+func (c *guestfsCommand) run(args []string) error {
```

Comment:
If you revert this change and ignore the `*cobra.Command` with `_` you can pass `c.run` directly to `RunE` in L76 without the wrapping lambda func.